### PR TITLE
Draft code for sunburst migration [DO NOT MERGE]

### PR DESCRIPTION
**Pull request by SallyXue-0114**
sunburst feature + chakra playback in JavaScript!

### CODE REVIEW

File: csv2json.js
Comment (quinnouyang) [inline csv2json.js:1]: Fine for now, but for the sake of modularity let's not hard-code the chakra and cateogry together (`fullName`) and the tooltip (`label`).

File: script.js [hunk 1]
```diff
@@ -0,0 +1,236 @@
...
+
+  // Function to load and parse the CSV
+  function loadDiseaseData() {
+    d3.csv("/public/seq.d3.csv").then((data) => {
+      diseaseData = data;
+    });
+  }
+
+  // Call the function to load the data
+  loadDiseaseData();
```
Comment (quinnouyang) [inline script.js:92]: Unless I misunderstood, I'd rather not rely on the CSV just for row numbers to identify which audio file to play. We can encode this mapping in the JSON instead.

**Review by quinnouyang — COMMENTED** [review]
Mostly things to address after migrating, but to think about now

**Comment by SallyXue-0114** [comment]
[like]  Xue, Sally reacted to your message:
________________________________

**Comment by staylor7** [comment]
OMG, I still need to make the time indexes, my apologies! I can have that by tomorrow, sorry!
_________________________________
Stephen Taylor
Professor of Music
University of Illinois at Urbana-Champaign
co-director, Illinois Modern Ensemble
www.stephenandrewtaylor.net
217 333-3712 office

**Comment by quinnouyang** [comment]
@staylor7 Ah no rush, I haven't even gotten to that part yet. Plenty of work to do here!

**Pull request closed without merging**